efel/cppcore/LibV5.cpp

```diff
@@ -26,6 +26,7 @@
 #include <deque>
 #include <functional>
 #include <iterator>
+#include <iostream>
```

Comment:
Was iostream required while debugging?

Reply:
oops, I made a few cout while debugging and forgot to remove it. I'll remove it just wait a few secs

Reply:
I also do that often :))